File: visaa_prep/R_P_S.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        char vigneshThrow = sc.next().charAt(0);
        char charanThrow = sc.next().charAt(0);
        if (vigneshThrow == 'S' && charanThrow == 'P'|| vigneshThrow == 'P' && charanThrow == 'R' || vigneshThrow == 'R' && charanThrow == 'S') {
            System.out.println("Vignesh");
        } else if (charanThrow == 'S' && vigneshThrow == 'P' || charanThrow == 'P' && vigneshThrow == 'R' || charanThrow == 'R' && vigneshThrow == 'S'){
          System.out.println("Charan");
        } else { 
          System.out.println("NULL");
        }
   }
}

File: visaa_prep/R_P_S.py
Vignesh, Charan = input().split()
if (Vignesh == 'S' and Charan == 'P') or (Vignesh == 'P' and Charan == 'R') or(Vignesh == 'R' and Charan == 'S'):
    print("Vignesh")
elif (Charan == 'S' and Vignesh == 'P') or (Charan == 'P' and Vignesh == 'R') or (Charan == 'R' and Vignesh == 'S'):
    print("Charan")
else:
    print("NULL")
